### ISSUE TITLE
[Training Issue]

### ISSUE THREAD

**Issue by emailxuchen** [eric612/MobileNet-YOLO#156]
hi，eric

Model : mobilenet_yolov3_lite

Questions : 为什么用mobilenet_yolov3_lite训练自己准备的数据集时（所有图片已经resize到512*512），loss会很快地变得这么小，是因为mobilenet_yolov3_lite_train.prototxt里设置的anchor与数据集不匹配的原因吗？

**Comment by emailxuchen** [comment]
I0831 23:01:50.301103 24860 sgd_solver.cpp:121] Iteration 3460, lr = 0.0005
I0831 23:01:56.710445 24860 solver.cpp:253] Iteration 3470 (1.56029 iter/s, 6.40908s/10 iters), loss = 9.07848e-06
I0831 23:01:56.710533 24860 solver.cpp:272]     Train net output #0: det_loss1 = 5.1068e-15 (* 1 = 5.1068e-15 loss)
I0831 23:01:56.710546 24860 solver.cpp:272]     Train net output #1: det_loss2 = 7.21709e-15 (* 1 = 7.21709e-15 loss)
I0831 23:01:56.710556 24860 sgd_solver.cpp:121] Iteration 3470, lr = 0.0005
I0831 23:02:03.147022 24860 solver.cpp:253] Iteration 3480 (1.55371 iter/s, 6.4362s/10 iters), loss = 9.07848e-06
I0831 23:02:03.147116 24860 solver.cpp:272]     Train net output #0: det_loss1 = 1.62727e-14 (* 1 = 1.62727e-14 loss)
I0831 23:02:03.147130 24860 solver.cpp:272]     Train net output #1: det_loss2 = 9.03898e-15 (* 1 = 9.03898e-15 loss)
I0831 23:02:03.147158 24860 sgd_solver.cpp:121] Iteration 3480, lr = 0.0005
I0831 23:02:09.981894 24860 solver.cpp:253] Iteration 3490 (1.46317 iter/s, 6.83449s/10 iters), loss = 9.07848e-06
I0831 23:02:09.981990 24860 solver.cpp:272]     Train net output #0: det_loss1 = 3.58056e-14 (* 1 = 3.58056e-14 loss)
I0831 23:02:09.982005 24860 solver.cpp:272]     Train net output #1: det_loss2 = 1.87516e-14 (* 1 = 1.87516e-14 loss)
I0831 23:02:09.982017 24860 sgd_solver.cpp:121] Iteration 3490, lr = 0.0005
I0831 23:02:17.707340 24860 solver.cpp:253] Iteration 3500 (1.2945 iter/s, 7.72501s/10 iters), loss = 9.07848e-06
I0831 23:02:17.707665 24860 solver.cpp:272]     Train net output #0: det_loss1 = 3.7156e-15 (* 1 = 3.7156e-15 loss)
I0831 23:02:17.707693 24860 solver.cpp:272]     Train net output #1: det_loss2 = 2.77091e-15 (* 1 = 2.77091e-15 loss)
I0831 23:02:17.707707 24860 sgd_solver.cpp:121] Iteration 3500, lr = 0.0005
I0831 23:02:24.845050 24860 solver.cpp:253] Iteration 3510 (1.40113 iter/s, 7.13709s/10 iters), loss = 9.07848e-06
I0831 23:02:24.845151 24860 solver.cpp:272]     Train net output #0: det_loss1 = 3.73053e-15 (* 1 = 3.73053e-15 loss)
I0831 23:02:24.845165 24860 solver.cpp:272]     Train net output #1: det_loss2 = 1.88528e-15 (* 1 = 1.88528e-15 loss)
I0831 23:02:24.845188 24860 sgd_solver.cpp:121] Iteration 3510, lr = 0.0005
I0831 23:02:31.766518 24860 solver.cpp:253] Iteration 3520 (1.44486 iter/s, 6.92107s/10 iters), loss = 9.07848e-06
I0831 23:02:31.766607 24860 solver.cpp:272]     Train net output #0: det_loss1 = 1.35855e-14 (* 1 = 1.35855e-14 loss)
I0831 23:02:31.766620 24860 solver.cpp:272]     Train net output #1: det_loss2 = 2.79254e-15 (* 1 = 2.79254e-15 loss)
I0831 23:02:31.766644 24860 sgd_solver.cpp:121] Iteration 3520, lr = 0.0005
I0831 23:02:38.568455 24860 solver.cpp:253] Iteration 3530 (1.47025 iter/s, 6.80155s/10 iters), loss = 9.07848e-06
I0831 23:02:38.568544 24860 solver.cpp:272]     Train net output #0: det_loss1 = 1.1553e-14 (* 1 = 1.1553e-14 loss)
I0831 23:02:38.568557 24860 solver.cpp:272]     Train net output #1: det_loss2 = 1.53407e-14 (* 1 = 1.53407e-14 loss)
